arrow-site/docs/docs/fx/README.md

```diff
@@ -11,130 +11,91 @@ Arrow Fx is a functional companion to KotlinX Coroutines augmenting its api with
-import kotlinx.coroutines.*
-import arrow.fx.coroutines.*
-
-//sampleStart
-suspend fun sayHello(): Unit =
-  println("Hello World")
+## Maven Setup
 
-suspend fun sayGoodBye(): Unit =
-  println("Good bye World!")
-
-suspend fun greet(): Unit {
-  sayHello()
-  sayGoodBye()
-}
-
-fun main(): Unit = runBlocking { // The edge of our world
-  greet()
-}
-//sampleEnd
+Add the dependencies that you want to use:
+```xml
+<dependency>
+    <groupId>io.arrow-kt</groupId>
+    <artifactId>arrow-core</artifactId>
+    <version>0.13.2</version>
+</dependency>
 ```
 
-Arrow Fx emphasizes the guarantee that users understand when they are performing side effects in their program declaration.
-
-If you're not very familiar with Functional Programming, and you've made it this far, you may realize that, despite the buzzwords and some FP jargon, you already know how to use Arrow Fx in general.
-This is because Arrow Fx brings the most popular imperative style to effectful programs with few simple primitives for effect control and asynchronous programming.
-
-# Conclusion
-
-Arrow Fx offers an idiomatic way of doing effecfull FP with Kotlin's coroutine system.
-It does this by providing support for cancellation, error handling, resource handling and all goodies you might be familiar with from other functional effect systems.
+### Maven BOM file
+
+To avoid specifying the Arrow version for every dependency, a BOM file is available:
+
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>io.arrow-kt</groupId>
+      <artifactId>arrow-stack</artifactId>
+      <version>${arrow.version}</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+```
 
-Arrow Fx offers direct style syntax and effect control without compromises, and removes the syntactic burden of type parametrization while still yielding programs that are pure, safe, and referentially transparent.
+## Integrating with 3rd-party libraries
 
-Despite some limitations of the Kotlin type system, like lack of Higher Kinded Types, the Kotlin language is excellent for encoding typed programs with a first-class imperative syntax for FP.
-These programs are approachable by the broader programming community, and arguably easier to encode than some solutions currently used in mainstream Kotlin and Scala FP communities.
+Arrow Fx integrates with KotlinX Coroutines Fx, Reactor framework, and any library that can model effectful async/concurrent computations as `suspend`.
 
-Come check out Arrow fx and join the discussion in the [Kotlin Slack](https://kotlinlang.slack.com/messages/C5UPMM0A0) and [Gitter](https://gitter.im/arrow-kt/Lobby)
+If you are interested in the Arrow Fx library, please contact us in the main [Arrow Gitter](https://gitter.im/arrow-kt/Lobby) or #Arrow channel on the official [Kotlin Lang Slack](https://kotlinlang.slack.com/messages/C5UPMM0A0) with any questions and we'll help you along the way.
```

Comment:
```suggestion
If you are interested in the Arrow Fx library, please contact us in the #Arrow channel on the official [Kotlin Lang Slack](https://kotlinlang.slack.com/messages/C5UPMM0A0) with any questions and we'll help you along the way.
```
We should consider dropping gitter from all places as it just fragments our community which is active in Slack.

Reply:
Yes, 100% agreed. Trying to gather everyone in Slack would be best. I have an easy time keeping my eyes on Slack, but multiple channels that I don't actively use is cumbersome.